variables.tf

```diff
@@ -89,18 +89,38 @@ variable "config_flux" {
 }
 
 variable "config_velero" {
-  description = "Velero Configuration"
+  description = <<EOT
+    Configures velero and the velero bucket. An external velero bucket that 
+    is managed externally from this module can be passed in via 
+    `config_bucket = {id = "123"}`. If `config_bucket = {enable = true}` 
+    even with `install = false` the bucket will remain created.
+  EOT
 
   type = object({
-    install          = optional(bool)
-    version          = optional(string)
-    bucket           = optional(string)
+    install   = optional(bool)
+    version   = optional(string)
+    bucket_id = optional(string)
```

Comment:
Is this for an existing bucket_id? 🤔 Let's call it `existing_bucket_id`. But they should still have to set `config_bucket.enable = false` if it's provided to *not* create the bucket. But we use this one if provided for actual backups. If the intent is that `config_bucket.id` is an existing bucket, the name should still give context, so `config_bucket.existing_id` or something and we can delete this?